### PR TITLE
Create Platforms “hpe-alletra/home”

### PR DESCRIPTION
**Pull request by datamattsson**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by datamattsson** [comment]
Please don't merge until management approval.

File: content/platform/hpe-alletra/home.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<div class="w3-container w3-center w3-margin-bottom">
+  <a href="https://hackshack.hpedev.io/workshops"><button type="button" class="w3-btn w3-teal">Try now!</button></a>
+</div>
+
+---
+
+## Any questions on HPE Alletra?
+
+Join the [HPEDEV Slack Workspace](https://slack.hpedev.io/) and start a discussion in the [\#alletra](https://hpedev.slack.com/) channel.
```
Comment (choukden) [inline content/platform/hpe-alletra/home.md:56]: Hi Michael,
I have created the #alletra channel in HPE DEV workspace: 
https://hpedev.slack.com/archives/C025D75HHGC

You may want to set the Hotlink for #alletra accordingly.

**Review by choukden — COMMENTED** [review]
Hi Michael,

Thank you very much for your "Platform" contribution. We will merge it once you get the management approval. 
We will have to list the blog posts (probably your last two blog posts published on June 15) you may want to tie with the new Alletra platform and I will update those blog posts with the tag "hpe-alletra".

Thanks
Denis

File: content/platform/hpe-alletra/home.md (hunk 1 above)
Comment (datamattsson) [reply content/platform/hpe-alletra/home.md]: Done!

**Comment by datamattsson** [comment]
Please merge at your earliest, this rendering of the platform page has been approved by product management.

**Pull request merged**